tortuga_kits/gceadapter/events/listeners/scalesets.py

```diff
@@ -144,7 +146,6 @@ def run(self, event: ResourceRequestCreated):
         except Exception as ex:
             logger.error("Error creating resource request: %s", ex)
             self._store.delete(ssr.id)
-            raise
```

Comment:
When exceptions are raised from event listeners, those exceptions are captured and emitted as error events (https://github.com/UnivaCorporation/tortuga/blob/master/src/installer/src/tortuga/tasks/task.py), which in turn are created as log messages in the controller which can be viewed in the UI. If we do not raise an exception and only print it to the log, the UI will not report the error.

Reply:
Makes sense, I've added that line back in.